internal/store/postgres/migrations/000016_update_assets_versions.up.sql

```diff
@@ -0,0 +1 @@
+UPDATE assets_versions SET version = '0.1' WHERE version = '';
```

Comment:
Instead of modifying the db for it, can we handle it in go? Like if no version, assume 0.1?

Reply:
Getting the asset by version will be an issue in that case imo. Currently we're unable to get an asset's first version.

Reply:
In that case, instead of running migrations for version 0.1, I can add an extra case to query depending on the version number with:
```diff
+      if version == "0.1" {
+      ast, err := r.getByVersion(ctx, id, version, r.GetByID, sq.Eq{
+      sq.Eq{"a.urn": urn},
+      sq.IsNull{"a.version"},
+      } 
+       else{
	ast, err := r.getByVersion(ctx, id, version, r.GetByID, sq.Eq{
		"a.asset_id": id,
		"a.version":  version,
	})
+	}
```

Should we proceed with this?